lib/fathead/android_api_docs/parse.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin env python
+# -*- coding: utf-8 -*-
+from bs4 import BeautifulSoup
+from glob import glob
+import re
+
+def build_article(tr):
+    '''Builds fathead article entry.
+
+    '''
+
+    title = tr.td.a.contents[0]
+    url = (tr.find('a')['href'])
+    #Some packages don't have a description, but simply a page listing classes and interfaces (possibly enums)
+    try:
+        abstract = tr.find('p').contents[0]
+    except:
+        abstract = "More information about interfaces and classes at " + url
+    title = '<span class="prog__sub">%s</span>' % title
```

Comment:
@TM37 you don't need any HTML around the title -- it should be left as plaintext; You can remove this line 👍 

The title is the text that we check the query against for an identical match. i.e. we take the query and use it as a key for a lookup in our database of Fatheads. Does this make sense now?